### PR TITLE
FINERACT-2092: Payment Waiver - UI changes

### PR DESCRIPTION
**Pull request by somasorosdpc**
## Description
UI changes to introduce Payment Waiver repayment type and Advanced payment allocation for loan products.

## Related issues and discussion
#{FINERACT-2092}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Review by adamsaghy — APPROVED** [review]
LGTM

File: src/app/loans/loans-view/loan-accounts-button-config.ts [hunk 1]
```diff
@@ -133,6 +133,11 @@ export class LoansAccountButtonConfiguration {
             icon: 'coins',
             taskPermissionName: 'CREATE_GOODWILL_TRANSACTION',
           },
+          {
+            name: 'Payment Waiver',
+            icon: 'coins',
+            taskPermissionName: 'CREATE_GOODWILL_TRANSACTION',
```
Comment (alberto-art3ch) [inline src/app/loans/loans-view/loan-accounts-button-config.ts:139]: Please fix this permission for Payment Waiver

File: src/assets/translations/en-US.json [hunk 2]
```diff
@@ -2414,6 +2414,7 @@
             "Navigation": "Navigation",
             "Notifications": "Notifications",
             "Organization": "Organization",
+            "Payment Waiver": "Payment Waiver",
```
Comment (alberto-art3ch) [inline src/assets/translations/en-US.json:2417]: Please add the other translations for the rest of the languages supported

**Review by alberto-art3ch — COMMENTED** [review]
Please review my comments

File: src/assets/translations/sw-SW.json [hunk 3]
```diff
@@ -2413,7 +2413,8 @@
       "Navigation": "Urambazaji",
       "Notifications": "Arifa",
       "Organization": "Shirika",
-      "Payments": "Malipo",
```
Comment (adamsaghy) [inline src/assets/translations/sw-SW.json:2416]: I reckon here you have modified the wrong translation key!

**Review by adamsaghy — CHANGES_REQUESTED** [review]
Please kindly check the review comments!

File: src/assets/translations/en-US.json (hunk 2 above)
Comment (somasorosdpc) [reply src/assets/translations/en-US.json:2417]: done.

File: src/app/loans/loans-view/loan-accounts-button-config.ts (hunk 1 above)
Comment (somasorosdpc) [reply src/app/loans/loans-view/loan-accounts-button-config.ts]: fixed.

File: src/assets/translations/sw-SW.json (hunk 3 above)
Comment (somasorosdpc) [reply src/assets/translations/sw-SW.json:2416]: fixed

**Review by alberto-art3ch — APPROVED** [review]
LGTM

**Review by adamsaghy — APPROVED** [review]
LGTM

**Pull request merged**